.github/workflows/maven.yml

```diff
@@ -18,10 +18,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
-    - name: Set up JDK 8
+    - name: Set up JDK 17
       uses: actions/setup-java@v4
       with:
-        java-version: '8'
+        java-version: '17'
```

Comment:
XDocReport is working with Java 8 , why you need Java 17?

Reply:
Java 17 is required for OpenPDF 2.0.0 or later. The build for this pull request fails with java 8. If we want to upgrade openpdf, I don't think there is another way than to upgrade the minimum java requirement for XDocReport. Of course I'm not sure if losing java 8 support is acceptable for this project.

Reply:
I added the changes that would be needed to run the tests with jdk17.

Reply:
Sorry but the people staying on Java 8 just have to use the old Version of those libraries. For those who keep up to date there should be a release for OpenPDF 2.
You also could make separate Releases for Java 8

Reply:
> Sorry but the people staying on Java 8 just have to use the old Version of those libraries.

You speak about PDF converter, but XDocReport provides another feature like report generation.